documentation.html

```diff
@@ -361,11 +361,100 @@ <h2 class="rg-booky-title">Text and Icon Badge</h2>
+                                class="code-snip" loading="lazy" sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+                        </div>
+                    </div>
+                    <div class="mg-t-40">
+                        <h2 class="rg-booky-title">Rounded Image</h2>
+                        <div class="center-block">
+                            <div class="mg-t-30">
+                                <div class="round-img-container">
+                                    <img class="round-img img-resp" src="./Assets/Images/avatar-img.jpg"
+                                        alt="round image">
+                                </div>
+                            </div>
+                        </div>
+                        <div class="code-example">
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28189%2C213%2C234%2C0%29&t=vscode&wt=sharp&l=auto&width=859&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=18px&lh=144%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522round-img-container%2522%253E%250A%2509%253Cimg%2520class%253D%2522round-img%2520img-resp%2522%2520src%253D%2522.%252FAssets%252FImages%252Favatar-img.jpg%2522%2520alt%253D%2522round%2520image%2522%253E%250A%253C%252Fdiv%253E"
+                                class="code-snip" loading="lazy" sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+                        </div>
+                    </div>
+                </div>
+
+                <div id="input">
+                    <h1 class="md-title">Input</h1>
+                    <p class="rg-p mg-t-10">
+                        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Cupiditate architecto, esse
+                        consequuntur delectus vitae rerum corrupti consequatur aut est id dignissimos qui dolorum
+                        exercitationem facere eum eaque, porro commodi nihil.
+                    </p>
+                    <div class="mg-t-40">
+                        <h2 class="rg-booky-title">Example of Inputs</h2>
+                        <div class="center-block inp-ex">
+                            <div>
+                                <label class="input-label sm-title" for="name">Name</label>
+                                <input class="user-input" id="name" type="text" />
+                            </div>
+                            <div class="mg-t-10">
+                                <label class="input-label sm-title" for="name">Password</label>
+                                <input class="user-input" id="name" type="password" placeholder="6+ characters" />
```

Comment:
You have used same id="name" in password and in its label, can change it to password.